config/servicemappings/compute.yaml

```diff
@@ -2474,12 +2474,15 @@ spec:
           description: |-
             A reference to the CertificateMap resource uri that identifies a
             certificate map associated with the given target proxy. This field
-            can only be set for global target proxies.
+            can only be set for global target proxies. This field is only supported
+            for EXTERNAL and EXTERNAL_MANAGED load balancing schemes.
+            For INTERNAL_MANAGED, use certificate_manager_certificates instead.
```

Comment:
Nit: we use field name in the format of camel case: `certificateManagerCertificates`.